{{cookiecutter.project_slug}}/apps/core/management/commands/runserver.py

```diff
@@ -3,9 +3,8 @@
 import signal
 import subprocess
 
-from django.contrib.staticfiles.management.commands.runserver import (
+from django.contrib.staticfiles.management.commands.runserver import \
```

Comment:
Because of https://github.com/timothycrosley/isort/issues/466.

Hope they'll fix it at some point, but for now isort does 99% of the rest which we're happy with.